### DISCUSSION TITLE
Version 0.5.0 (final)

### DISCUSSION THREAD

**Discussion by DSheirer** [DSheirer/sdrtrunk#1389] [Announcements]
## **Application download links for each operating system are available below in the Assets section.**

**WARNING**: if you use Version 0.5.0 Beta 1 (or later) and create playlists with NBFM Squelch Control channels, these are not backward compatible with earlier versions and you risk corrupting your playlist.  Use separate playlists for your NBFM Squelch channels if you plan to use previous versions of sdrtrunk.

## 1. Features:
* NBFM Squelch

## 2. Requirements:
* Windows x86 (64-bit)
* Linux x86/aarch64 (64-bit)
* OSX x86/aarch64 (64-bit)

**Note**: JMBE library must be manually downloaded/built for OSX aarch64 due to OSX sandboxing of downloaded code.
**Note**: Linux for arm-32 is NOT currently supported

## 3. Optional Requirements
**JMBE version 1.0.9+** is required for decoding APCO25 Phase 1/2 and DMR digital audio on Linux and OSX aarch64 and this library can be compiled within the sdrtrunk application.

Legacy instructions for compiling the JMBE library are located at: **https://github.com/DSheirer/sdrtrunk/wiki/JMBE**

## 4. User Manual
* [Wiki](https://github.com/DSheirer/sdrtrunk/wiki)
* [User Manual](https://github.com/DSheirer/sdrtrunk/wiki/User-Manual)

## 5. Support
* [Support](https://github.com/DSheirer/sdrtrunk/wiki/Support)

## 6. Enhancements/Fixes

@doug-hoffman Correctly identify padded MP3 frames
#1237 MP3 encoder - prevent unsupported MP3 settings and input sample rates
#1287 Disable event panel filtering until feature is fully implemented
#1312 Radio reference system import - error for sites with no assigned County
#1378 USB Tuner shutdown can hang and crash application
#1380 Icecast streaming status error messages
#1383 Tuner editor buttons disabled on enabling a tuner

<hr /><em>This discussion was created from the release <a href='https://github.com/DSheirer/sdrtrunk/releases/tag/'>Version 0.5.0 (final)</a>.</em>

**Comment by Ulti-P-Uszer** [comment]
Thanks for the great Christmas gift @DSheirer of upgrading the program.
